dna/src/commonMain/kotlin/predicate/PredicateFunctionContractStrImpl.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.trilobiitti.trilobite.dna.predicate
+
+import com.github.trilobiitti.trilobite.dna.data.DataType
+
+class PredicateFunctionContractStrImpl: PredicateFunctionContract {
```

Comment:
Also, let's group all contracts into a single module (e.g. `StandardContracts.kt`) and make `predicate` package a subpackage of `data` package